File: Python_Programming/Practice/Addition.py
def Addition(num1, num2):

    return num1 + num2


def main():

    print("Enter first number : ")
    Value1 = int(input())

    print("Enter second number : ")
    Value2 = int(input())

    Ans = Addition(Value1, Value2)

    print("Addition is : ",Ans)

if __name__ == "__main__":
    main()

File: C_Programming/Practice/Addition.c
#include<stdio.h>
int Addition(int No1, int No2)
{
    int result = 0;
    result = No1+No2;
    return result;
}
int main()
{
    int i=0, j=0, ans=0;
    printf("Enter first number:\n");
    scanf("%d",&i);

    printf("Enter second number:\n");
    scanf("%d",&j);

    ans= Addition(i,j);
    printf("Addition is %d",ans);
    return 0;
}